### ISSUE TITLE
Upgrade file-type to 16.5.4

### ISSUE THREAD

**Issue by ghost** [strikeentco/mime-kind#12]
This module currently depends on a very old major version of `file-type`. While `file-type` 17 and newer are ESM-only, version 16 isn't and 16.5.4 fixes a CVE: https://nvd.nist.gov/vuln/detail/CVE-2022-36313

Can you please upgrade the dependency?

**Comment by strikeentco** [comment]
Hello!
It looks like, a `file-type` has changed a lot in the years since the last `mime-type` update.. :D

I will update a `multi-part` later this week. And probably, a `mime-type` will be deprecated.

Thanks!)

**Comment by strikeentco** [comment]
Hello, @pluma4345 !

Just released new `4.0.0` versions of `mime-kind` and `multi-part`.

There is only one breaking change: there are no more `sync` versions.

For `arangojs` it means, that you should change `toForm` function: https://github.com/arangodb/arangojs/blob/main/src/lib/multipart.ts#L38
to something like this:
```
export async function toForm(fields: Fields): Promise<MultipartRequest> {
  const form = new Multipart();
  for (const key of Object.keys(fields)) {
    let value = fields[key];
    if (value === undefined) continue;
    if (
      !(value instanceof Readable) &&
      !(value instanceof global.Buffer) &&
      (typeof value === "object" || typeof value === "function")
    ) {
      value = JSON.stringify(value);
    }
    form.append(key, value);
  }
  const body = await form.buffer();
  return { body, header: form.getHeaders(false) };
}
```
That's it! :)

Thank you.

**Comment by ghost** [comment]
Hi @strikeentco! Thanks for the prompt reaction and the detailed suggestion. I'll incorporate the changes in the next release.
